xamarin/Xamarin.Forms
# [UWP] Prevents Picker collapsing on opening Dropdown menu

# Description of Change ###
Changes the behavior of the Picker on UWP by preventing the UWP Combobox from altering it's width when the drop down opens.

I considered trying to retain the UWP MinWidth of the control, but it doesn't seem to be used.



### Issues Resolved ### 


- fixes #6403 

### API Changes ###
 None

### Platforms Affected ### 


- UWP

### Behavioral/Visual Changes ###
Prevents the UWP combo box from shrinking to min size when the drop down opens and a width is not set.

### Before/After Screenshots ### 


Before
![58880478-40d0e280-86d8-11e9-9163-e31f52f656d3](https://user-images.githubusercontent.com/15127788/67358207-4747e580-f525-11e9-90cb-c533b71ec617.gif)

After
![After](https://user-images.githubusercontent.com/15127788/67359975-6137f700-f52a-11e9-86dd-a17bfdcb45a6.gif)

### Testing Procedure ###
Open the issue page and select various items from the picker.  Ensure that the label to the left does not move on the drop down open, only after selecting an item.

### PR Checklist ###
